enterprise/micronaut/src/org/netbeans/modules/micronaut/hyperlink/HyperlinkImpl.java

```diff
@@ -0,0 +1,290 @@
+/*
```

Comment:
I do not like location of this file inside of `enterprise/micronaut` module. I would prefer leaving this file as a part of `ide/editor` module and adding its dependency on `ide/api.lsp` if necessary.

Reply:
Removed in ee9eb43